frontend/SplitPane.js

```diff
@@ -140,6 +140,7 @@ const styles = {
   rightPane: {
     display: 'flex',
     width: '100%',
+    overflow: 'auto',
```

Comment:
The relative positioned drag handle was causing weird overflow behavior in this component in some cases. This style fixes it though!